bin/BuildPackages.sh

```diff
@@ -236,17 +209,13 @@ build_one_package() {
       build_carat
     ;;
 
-    cohomolo*)
```

Comment:
Not only is it not needed, the special code failed to work because there is no `makefile.orig` in `cohomolo`.